lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +93,16 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+
+          if is_loopback_address(ip)
```

Comment:
Just pushed up the fix. Sorry for the confusion! I also kept the `is_loopback_address` method at the module level, that's a lot DRY'er than what I had before. Cheers!